test/functional/wallet_createwallet.py

```diff
@@ -96,5 +96,22 @@ def run_test(self):
         assert_raises_rpc_error(-4, "Error: This wallet has no available keys", w5.getnewaddress)
         assert_raises_rpc_error(-4, "Error: This wallet has no available keys", w5.getrawchangeaddress)
 
+        self.log.info('New blank and encrypted wallets can be created')
+        self.nodes[0].createwallet(wallet_name='wblank', disable_private_keys=False, blank=True, passphrase='thisisapassphrase')
+        wblank = node.get_wallet_rpc('wblank')
+        wblank.walletpassphrase('thisisapassphrase', 10)
```

Comment:
Suggest adding the following tests:

- test that the wallet is locked by calling a method that requires the wallet to be unlocked (eg `signmessage`) and testing that it triggers an error
- test that there are no keys in the wallet

Reply:
Done